go.mod

```diff
@@ -1,11 +1,12 @@
-module github.com/ProtonMail/gopenpgp
+module github.com/ProtonMail/gopenpgp/v2
```

Comment:
now I see.. OK.. so it seems that both options `.v2` and `/v2` are valid

I would expect that:
* `/v2` means that our repo has folder `/v2`
* `.v2` means you don't have the folder and you will not continue with `v1`

I don't think it's a rule so feel free to ignore all those comments